2-structured-data/test/e2e/CloudSqlTest.php

```diff
@@ -22,9 +22,9 @@
  */
 class CloudSqlTest extends E2eTest
 {
-    protected static function setBackEnd()
+    protected static function getCustomConfig()
     {
-        putenv('BOOKSHELF_BACKEND=cloudsql');
+        return ['bookshelf_backend' => 'cloudsql'];
```

Comment:
@tmatsuo please note the change in how e2e tests work - the setting of the environment variables was having an adverse effect on the tests.